R/snapshot.R

```diff
@@ -23,8 +25,14 @@
 #'
 #' head(read.csv(tmp))
 
-snap <- function(to = "packages.csv") {
-  pkgs <- installed.packages(priority = NA_character_)
+snap <- function(to = "packages.csv", keepRecommended=FALSE) {
+  
+  priority <- NA_character_
+  if(keepRecommended) {
+    priority <- c(NA_character_, "recommended")
+  }
```

Comment:
Agreed. Although this will *only* bring in recommended whereas I think we want recommended *and* user packages. So something like
```r
priority <- if (recommended) "recommended" else c( "recommended", NA_character_)
```